js/deer-config.js

```diff
@@ -161,7 +161,12 @@ export default {
                 </style>
                 <h2> Glosses </h2>
                 <small class="cachedNotice is-hidden text-primary"> These Glosses were cached.  To reload the data <a class="newcache tag is-small">click here</a>. </small>
-                <input filter="title" type="text" placeholder="&hellip;Type to filter by incipit, text, or targeted text" class="is-hidden serifText">
+				<form id="ngForm" deer-type="Gloss" deer-context="http://www.loc.gov/mods" class="row">
```

Comment:
I'm not certain why this is a form. There isn't a need for a creator or anything that I can see, just processing the search term. Let me know if I'm missing something.